src/DynamoUtilities/DynamoFeatureFlagsManager.cs

```diff
@@ -73,6 +73,13 @@ internal void CacheAllFlags()
             }
         }
 
+        /// <summary>
+        /// Check feature flag value, if not exist, return defaultval
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="featureFlagKey"></param>
+        /// <param name="defaultval"></param>
+        /// <returns></returns>
```

Comment:
I found there was no description when trying to the call, so I gave it a try, let me know if you would like me to add more here